lib/tooling/pahole-tool.ts

```diff
@@ -33,7 +33,7 @@ export class PaholeTool extends BaseTool {
 
     override async runTool(compilationInfo: Record<any, any>, inputFilepath?: string, args?: string[]) {
         if (!compilationInfo.filters.binary && !compilationInfo.filters.binaryObject) {
-            return this.createErrorResponse('Pahole requires a binary output');
+            return this.createErrorResponse(`${this.tool.name ? this.tool.name : 'Pahole'} requires an executable`);
```

Comment:
```suggestion
            return this.createErrorResponse(`${this.tool.name || 'Pahole'} requires an executable`);
```
(fly by review only! lgtm either way)